File: TransCoder_tokenized_test_set_functions/java/FIND_ONE_EXTRA_CHARACTER_STRING_1.java
static char findExtraCharcter ( String strA, String strB ) {
  int res = 0, i;
  for ( i = 0;
  i < strA . length ( );
  i ++ ) {
    res ^= strA . charAt ( i );
  }
  for ( i = 0;
  i < strB . length ( );
  i ++ ) {
    res ^= strB . charAt ( i );
  }
  return ( ( char ) ( res ) );
}

File: results/py/FIND_ONE_EXTRA_CHARACTER_STRING_1.cpp
char f_filled ( string strA , string strB ) {
  int res = 0 ;
  for ( int i = 0 ;
  i < strA . length ( ) ;
  i ++ ) res = res ^ ( ( int ) strA [ i ] ) ;
  for ( int i = 0 ;
  i < strB . length ( ) ;
  i ++ ) res = res ^ ( ( int ) strB [ i ] ) ;
  return ( ( char ) res ) ;
}
